02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,46 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Killrilla: Gorilla Extermination Services</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
 
+  <h1>
+    <img src="http://weeklyworldnews.files.wordpress.com/2011/01/target-midstory.jpg" alt="Killrilla Logo" height="50px" width="50px"/>Killrilla
+  </h1>
+  <h3>Gorilla Extermination Services</h3>
+
+  <div style="background-color:90D492">
```

Comment:
for future reference - we are not going to have more in-line style in our projects